src/stripe.common.ts

```diff
@@ -47,6 +47,20 @@ export interface Token {
   livemode: boolean;
 }
 
+export interface Source {
+  amount: number;
```

Comment:
Minor, but I find it useful to add a comment to the end of this line
`// in pennies`

The same should be added in `index.d.ts` to keep them consistent. I actually think most people using the plugin will look first in `index.d.ts` so that one is more important.